docs/docsite/rst/playbook_guide/playbooks_startnstep.rst

```diff
@@ -19,6 +19,14 @@ To start executing your playbook at a particular task (usually the task that fai
 
 In this example, Ansible starts executing your playbook at a task named "install packages". This feature does not work with tasks inside dynamically re-used roles or tasks (``include_*``), see :ref:`dynamic_vs_static`.
 
+.. note::
+
+    Moreover, ``--start-at-task``
+    - can target tasks **within** static includes
```

Comment:
`within imports (which are static)` .. dont say 'static includes' as that would be a contradiction in our terminology, in which all includes are dynamic

Reply:
same with below, just establish import == static/preprocessed, include ==dynamic/executed at runtime